tests/test_commands.py

```diff
@@ -642,6 +642,16 @@ def test_dump_and_restore_and_replace(self, r):
         r.restore('a', 0, dumped, replace=True)
         assert r['a'] == b'bar'
 
+    @skip_if_server_version_lt('5.0.0')
+    def test_dump_and_restore_absttl(self, r):
+        r['a'] = 'foo'
+        dumped = r.dump('a')
+        del r['a']
+        ttl = int(time.time() * 1000) + 2000
```

Comment:
This should possibly use `r.time()`, otherwise the test may fail if there is clock drift between the server and the test machine.